src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/RssLink.java

```diff
@@ -11,7 +11,7 @@
 
     public RssLink(@NonNull Blog blog) {
         Preconditions.checkArgument(StringUtils.isNotBlank(blog.getRss()),
-                "Rss link can not be null");
+                "Rss link can not be NULL or empty.");
```

Comment:
It's gramatically incorrect, sgould be: _Rss link ~~can not~~ cannot be NULL ~~or~~ nor empty._